.github/workflows/ecosystem.yml

```diff
@@ -229,7 +230,7 @@ jobs:
           - geometric
           - openfermion>=1.0
           - openfermionpsi4
-          - pyddx
+          - pyddx=0.3.0
```

Comment:
I made a comment on this exact change in https://github.com/psi4/psi4/pull/2921, and I'll repeat it here.

"A quick thought on my end, since we're enforcing pyddx 0.3.0 here for now. In the ddx/CMakeLists.txt file in the external/upstream/ directory, there is a line for finding the pyddx Python module:

`find_python_module(pyddx ATLEAST 0.3.0 QUIET)`

Would it be necessary, or even desirable, to temporarily switch this to

`find_python_module(pyddx EXACT 0.3.0 QUIET)`

to simply not allow pyddx v0.4 to be used with Psi4 at the moment?"

Reply:
Yeah, sorry, I made the change here just to get the PR working before I noticed the comment on the other PR.

the `ATLEAST` to `EXACT` is a good notion (and a vital one if the restriction hangs around until the release). I won't close #2921 without that change, but I don't think it need hold up this one.

Reply:
No worries! And that's fair, to be honest. We can worry about this issue in that PR.